orchestration/pyproject.toml

```diff
@@ -6,26 +6,28 @@ authors = ["Your Name <you@example.com>"]
 
 [tool.poetry.dependencies]
 # to prevent unininteded upgrades we need to provide upperbound versions for the dependencies. For example ~3.4 instead of ^3.4
-python = "3.9.12"
+python = "3.9.16"
 argo-workflows = "^5.0.0"
+broad-dagster-utils = "0.6.7"
 cached-property = "^1.5.2"
 dagster = "0.12.14"
+dagster-gcp = "^0.12.14"
 dagster-k8s = "0.12.14"
 dagster-postgres = "0.12.14"
 dagster-slack = "0.12.14"
 data-repo-client = "^1.134.0"
 google-cloud-bigquery = "^2.4.0"
 google-cloud-storage = "^1.3.5"
-python-dateutil = "^2.8.1"
-typing-extensions = "^3.7.4"
-pyyaml = "^5.3"
-dagster-gcp = "^0.12.14"
-broad-dagster-utils = "0.6.7"
 graphql-ws = "<0.4.0"
 hca-import-validation = "^0.0.16"
-rfc3339-validator = "^0.1.4"
+mako = "1.2.2"
```

Comment:
What's this dependency for?

Reply:
[Looks like it's a dependency of Sqlalchemy](https://github.com/DataBiosphere/hca-ingest/security/dependabot/6)

Reply:
well technically it is a dependency of alembic which is a database migration tool for SQLAlchemy (no standard capitalization there I guess...?)